feeds/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "isolatedModules": true,
     "preserveSymlinks": true
   },
-  "include": ["src", "src/**/*.json", "@types"],
+  "include": ["src", "src/**/*.json", "@types", "src/assets/**/*.ts"],
```

Comment:
This should be included by `"src"` already

Reply:
Yes, this is what i was thinking but then i had an issue with the line:
```
import { sponsorList, SponsorListItem } from '../../assets/sponsors'
```